data/templates/squid/sg_acl.conf.j2

```diff
@@ -1,7 +1,10 @@
 ### generated by service_webproxy.py ###
 dbhome {{ squidguard_db_dir }}
-
+{% if rule == '' %}
```

Comment:
Why not use: is not vyos_defined()?

Reply:
It was my mistake. Changes to this file were not needed.